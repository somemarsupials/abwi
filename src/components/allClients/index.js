export { default as ClientsPage } from './clientsList';
export { default as ClientsList } from './clientsPage';
