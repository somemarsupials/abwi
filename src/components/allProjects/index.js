export { default as List } from './list';
export { default as Page } from './page';
