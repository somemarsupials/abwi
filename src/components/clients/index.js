export { default as ClientPage } from './clientPage';
export { default as ClientDetail } from './clientDetail';
