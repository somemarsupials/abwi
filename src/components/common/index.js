export { default as TitleBar } from './titleBar';
export { default as ErrorBar } from './errorBar';
