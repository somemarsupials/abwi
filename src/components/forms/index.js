export { default as ProjectForm } from './projectForm';
