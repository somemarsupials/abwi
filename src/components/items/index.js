export { default as ItemList } from './itemList';
