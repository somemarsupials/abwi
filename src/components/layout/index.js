export { default as SubHeader } from './subHeader';
