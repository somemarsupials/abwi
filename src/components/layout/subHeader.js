import React from 'react';

export default function(props) {
  return (
    <h2 className="page-header">
      {props.children}
    </h2>
  );
};
