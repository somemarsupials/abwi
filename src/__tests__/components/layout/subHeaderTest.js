import React from 'react';
import { shallow } from 'enzyme';

import { SubHeader } from '../../../components/layout';

describe('Header', () => {
  let header;

  describe('when passed detailed information', () => {
    beforeEach(() => {
      header = shallow(<SubHeader>text</SubHeader>);
    });

    it('correctly renders', () => {
      expect(header).toMatchSnapshot();
    });
  });
});

