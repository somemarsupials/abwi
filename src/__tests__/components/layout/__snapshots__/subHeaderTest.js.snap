// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header when passed detailed information correctly renders 1`] = `
<h2
  className="page-header"
>
  text
</h2>
`;
